src/main/scala/io/iohk/ethereum/domain/Account.scala

```diff
@@ -1,21 +1,26 @@
 package io.iohk.ethereum.domain
 
 import akka.util.ByteString
-import org.spongycastle.util.encoders.Hex
+
+object Account {
+  val Empty = Account(0, 0, ByteString.empty, ByteString.empty)
```

Comment:
I got it from downloading met state tree for accounts that had empty storage (not contracts) I was getting this hash, I assume that it is hash of empty mpt node